core/services/job_subscriber.go

```diff
@@ -71,7 +71,7 @@ func NewJobSubscriber(store *store.Store, runManager RunManager) JobSubscriber {
 }
 
 func (js *jobSubscriber) Start() error {
```

Comment:
Wat. Can't we just delete this since it does absolutely nothing?

Reply:
Should be able to, yeah, although I want confirmation from @j16r first, if not others as well, that there aren't interfaces that jobSubscriber needs to conform to that require this.

Reply:
Lean on the compiler ;)